src/components/CalendarDay.jsx

```diff
@@ -55,20 +64,28 @@ export default class CalendarDay extends React.Component {
       isOutsideDay,
       modifiers,
       renderDay,
+      isFocused,
     } = this.props;
 
     const className = cx('CalendarDay', {
       'CalendarDay--outside': !day || isOutsideDay,
     }, getModifiersForDay(modifiers, day).map(mod => `CalendarDay--${mod}`));
 
     return (day ?
-      <td
-        className={className}
-        onMouseEnter={e => this.onDayMouseEnter(day, e)}
-        onMouseLeave={e => this.onDayMouseLeave(day, e)}
-        onClick={e => this.onDayClick(day, e)}
-      >
-        {renderDay ? renderDay(day) : day.format('D')}
+      <td className={className}>
+        <button
+          type="button"
+          ref={(ref) => { this.buttonRef = ref; }}
+          className="CalendarDay__button"
+          aria-label={`${day.format('dddd')}. ${day.format('LL')}`}
+          onMouseEnter={e => this.onDayMouseEnter(day, e)}
+          onMouseLeave={e => this.onDayMouseLeave(day, e)}
+          onMouseUp={e => e.currentTarget.blur()}
+          onClick={e => this.onDayClick(day, e)}
+          tabIndex={isFocused ? 0 : -1}
+        >
+          {renderDay ? renderDay(day) : day.format('D')}
```

Comment:
We originally decided my suggestion of using a component here like `<Day>{renderDay ? renderDay(day) : day.format('D')}</Day>` so that a propType could be applied to the return value - is that perhaps worth doing here?